src/app/home/home-router.module.ts

```diff
@@ -12,70 +12,75 @@ import { CommunityComponent } from '../community/community.component';
 import { myDashboardRoute } from './router-constants';
 import { CoursesProgressLearnerComponent } from '../courses/progress-courses/courses-progress-learner.component';
 import { LandingComponent } from '../landing/landing.component';
+import { AuthService } from '../shared/auth-guard.service';
+import { BetaThenAuthService } from '../shared/beta-then-auth-guard-service';
 
 export function dashboardPath(route): string {
   return `${myDashboardRoute}/${route}`;
 }
 
+const alwaysGuardedRoutes = [

```

Comment:
I can revert this if needed. I think this is a little more organized but also I see it is cluttering the diff.